apps/frontend/src/app/Data/Characters/Lyney/index.tsx

```diff
@@ -507,7 +492,27 @@ const sheet: ICharacterSheet = {
     constellation3: ct.talentTem('constellation3', [
       { fields: [{ node: nodeC3 }] },
     ]),
-    constellation4: ct.talentTem('constellation4'),
+    constellation4: ct.talentTem('constellation4', [
```

Comment:
Why move this?

Reply:
think it's easier to understand if it's there rather than under the Talent (understand that it's a Pyro CA, and any Pyro CA when there are two). Also less crowded. Though ig the first reason could be done to some degree with strings